internal/telemetry/client.go

```diff
@@ -389,48 +393,111 @@ func (c *Client) newRequest(t RequestType) *Request {
 			LanguageVersion: runtime.Version(),
 		},
 		Host: Host{
-			Hostname:    hostname,
-			ContainerID: internal.ContainerID(),
-			OS:          getOSName(),
-			OSVersion:   getOSVersion(),
+			Hostname:     hostname,
+			OS:           getOSName(),
+			OSVersion:    getOSVersion(),
+			Architecture: runtime.GOARCH,
+			// TODO (lievan): getting kernel name, release, version TBD
```

Comment:
Yea, I'll create a ticket for it - its a small change, but it's probably best to make it in a separate PR since getting kernel info is a bit more complicated compared to just importing some constant